biggus/__init__.py

```diff
@@ -2281,14 +2281,27 @@ def _full_keys(keys, ndim):
     if not isinstance(keys, tuple):
         keys = (keys,)
 
+    # Make keys mutable, and take a copy.
+    keys = list(keys)[:]
```

Comment:
As I wasn't 100% confident what the result of the following was:

```
>>> a = [1, 2, 3]
>>> b = list(a)
>>> a is b
False
```

As a consequence, the ``[:]`` is redundant.